### PR TITLE
Facebook Pixel Documentation Updates

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Updated the page section to specify that the page properties and the event ID are now automatically passed downstream.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: destinations/advertising/fb-pixel.mdx [hunk 1]
```diff
@@ -157,6 +157,12 @@ A sample `page` call is as shown:
 rudderanalytics.page()
 ```
 
+<div class="successBlock">
+
+ Once you make the <code class="inline-code">page</code>, RudderStack automatically sends the page properties and the event ID to Facebook Pixel.
```
Comment (akashrpo) [inline destinations/advertising/fb-pixel.mdx:162]: Line:152 We can actually pass properties to `page` now which is optional and by default it is empty `{}`. Also, can we re-write the statement
```
the `track` event is sent as `PageView` to `fbq('track,'PageView')
```
as follows
```
the SDK will send the data to Facebook Pixel with event type `pageView`.
```

Comment (saikumarrs) [reply destinations/advertising/fb-pixel.mdx:162]: @ameypv-rudder @akashrpo 
"Any parameter sent to `rudderanalytics.page()` is ignored by RudderStack." needs to be replaced with the content from the newly added "successBlock".

Comment (ameypv-rudder) [reply destinations/advertising/fb-pixel.mdx]: incorporated the changes. @akashrpo @saikumarrs

File: destinations/advertising/fb-pixel.mdx [hunk 2]
```diff
@@ -149,7 +149,12 @@ rudderanalytics.identify("userId", userVars) // userVars is a JSON object
 
 ## Page
 
-When the `page`call is made, the `track` event is sent as `PageView` to `fbq('track,'PageView')`. Any parameter sent to `rudderanalytics.page()` is ignored by RudderStack.
+When you make the `page` call, RudderStack sends the data to Facebook Pixel with the event type as `pageView`. You can also pass properties to `page` - RudderStack automatically sends these properties along with the event ID to Pixel.
+
+<div class="infoBlock">
+
+  Passing properties to <code class="inline-code">page</code> is optional. By default, it is empty.
+</div>
```
Comment (saikumarrs) [inline destinations/advertising/fb-pixel.mdx:157]: This seems redundant as it is understood from our event spec and the example below. Moreover, we don't specify this in page calls for any destinations

**Review by saikumarrs — CHANGES_REQUESTED** [review]

File: destinations/advertising/fb-pixel.mdx (hunk 2 above)
Comment (saikumarrs) [reply destinations/advertising/fb-pixel.mdx:157]: @akashrpo SDK adds default properties to the incoming properties object in the page call for the final rudder message object.

Comment (ameypv-rudder) [reply destinations/advertising/fb-pixel.mdx]: removed the tip.

**Review by saikumarrs — APPROVED** [review]

**Pull request merged**